src/main/java/com/bisnode/opa/client/OpaClient.java

```
@@ -0,0 +1,48 @@
+package com.bisnode.opa.client;
+
+
+import com.bisnode.opa.client.data.OpaDataApi;
+import com.bisnode.opa.client.data.OpaDataClient;
+import com.bisnode.opa.client.policy.OpaPolicyApi;
+import com.bisnode.opa.client.policy.OpaPolicyClient;
+import com.bisnode.opa.client.query.OpaQueryApi;
+import com.bisnode.opa.client.query.OpaQueryClient;
+import com.bisnode.opa.client.rest.OpaRestClient;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Delegate;
+import lombok.experimental.FieldDefaults;
+
+import java.net.http.HttpClient;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@FieldDefaults(makeFinal = true, level = PRIVATE)
```

If you have to use Lombok, please don't use `@FieldDefaults`, it hides the intention. Make fields `private final` instead.